packages/Kokkos/src/CMakeLists.txt

```diff
@@ -282,9 +282,9 @@ APPEND_SET(HEADERS ${DIR}/${PACKAGE_NAME}_ETIHelperMacros.h)
 # Explicitly instantiate classes.
 IF (${PACKAGE_NAME}_ENABLE_EXPLICIT_INSTANTIATION)
 
-  # Generate ETI .cpp files for DataTransferKit::PointCloud.
-  DTK_PROCESS_ALL_SLGN_TEMPLATES(POINTCLOUD_OUTPUT_FILES "DTK_ETI_SC_LO_GO_NT.tmpl" "PointCloud" "POINTCLOUD" "${${PACKAGE_NAME}_ETI_SCALARS}" "${${PACKAGE_NAME}_ETI_LORDS}" "${${PACKAGE_NAME}_ETI_GORDS}" "${${PACKAGE_NAME}_ETI_NODES}" TRUE)
-  LIST(APPEND SOURCES ${POINTCLOUD_OUTPUT_FILES})
+  # # Generate ETI .cpp files for DataTransferKit::Field.
+  # DTK_PROCESS_ALL_SLGN_TEMPLATES(FIELD_OUTPUT_FILES "DTK_ETI_SC_LO_GO_NT.tmpl" "Field" "FIELD" "${${PACKAGE_NAME}_ETI_SCALARS}" "${${PACKAGE_NAME}_ETI_LORDS}" "${${PACKAGE_NAME}_ETI_GORDS}" "${${PACKAGE_NAME}_ETI_POLYHEDRONS}" TRUE)
+  # LIST(APPEND SOURCES ${FIELD_OUTPUT_FILES})
```

Comment:
I guess we should really be doing ETI on these classes. Especially if DTK is being built as a separate library. I think we need to make the *.hpp files *_decl.hpp, add instantiation macros to a *_def.hpp, and add them to the build.